src/main/java/org/opentripplanner/datastore/FileType.java

```diff
@@ -45,7 +45,7 @@ public String text() {
    * At least one input data file must be present to build a graph.
    */
   public boolean isInputDataSource() {
```

Comment:
Updaate Java docs with ```@return``` description

Reply:
are you sure it should be touched? it is the original code from the otp2